static/app/components/replays/videoReplayer.spec.tsx

```diff
@@ -0,0 +1,227 @@
+import {VideoReplayer} from './videoReplayer';
+
+// XXX: Not quite sure the best way to mock RAF - here we use fake timers
+// VideoReplayer uses `app/util/replays/timer` which uses RAF to count up. This
+// is used to render the progress of the seeker bar and sync with video
+// replays.
+//
+// advancing by 2000ms ~== 20000s in Timer, but this may depend on hardware, TBD
+jest.useFakeTimers();
```

Comment:
cc @scttcper for the v18 check on timers

Reply:
this doesn't really look like react so its probably fine